pgx-examples/versioned_so/versioned_so.control

```diff
@@ -0,0 +1,4 @@
+comment = 'versioned_so:  Created by pgx'
+default_version = '@CARGO_VERSION@'
+relocatable = false
+superuser = false
```

Comment:
Just for clarity, add a commented out `module_pathname` property here, with some explanation as to what it means to **not** have defined.

Reply:
Also, please add a run line for the example in the appropriate place around here:  https://github.com/tcdi/pgx/blob/c4ce4b9a5d74d589345fe95f3175a9e656dc6b3f/.github/workflows/tests.yml#L135

Reply:
Also, also, please add the example to the top-level workspace here:  https://github.com/tcdi/pgx/blob/master/Cargo.toml

Reply:
Done. And added the example to the GitHub workflow.